docs/tutorial/part02.rst

```diff
@@ -1,6 +1,66 @@
-==========================
-Set the Neomodel database.
-==========================
+============================
+Set the NeoModel database
+============================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+On this section we are going to install Neo4j on our local machine and show how you going to connect it with our app.

```

Comment:
maybe add `are` after how you. `you are going to connect it`

Reply:
Done!!! Thanks!!!!